tests/integration/test_ipam.py

```diff
@@ -26,3 +27,14 @@ def test_prefixes_successfully_stringify_tags(nb_client):
     assert "192.0.2.0/24" in str(prefix)
     assert prefix.tags
     assert isinstance(prefix.tags[0], Record)
+
+
+@pytest.mark.timeout(120)
+def test_prefixes_pagination_with_max_page_size(nb_client):
+    """Validate prefixes are returned when dataset is larger than max_page_size and not in endless loop."""
+    for i in range(0, 256):
+        nb_client.ipam.prefixes.create(
+            prefix=f"192.1.{str(i)}.0/24", namespace={"name": "Global"}, status={"name": "Active"}
+        )
```

Comment:
Wouldn't this be better as a comprehension so it's just 1 POST call?

Reply:
of course!